src/components/LoginContext.js

```diff
@@ -0,0 +1,3 @@
+import { createContext } from 'react';
+
+export const LoginContext = createContext({});
```

Comment:
You can create defaults here, it will be easier to read what is expected from provider